### PR TITLE
upgrade srcery theme

### PR DESCRIPTION
**Pull request by redguardtoo**
upgrade srcery  theme

Update theme srcery with minor change, as I was enlisted in #3118

### Changes
Add code comment, adjust a few entity's color

### Checklist
- [x] they don't apply here because it's minor css change

- [x] Updated the changelog at `CHANGES.md`

### CODE REVIEW

**Comment by joshgoebel** [comment]
Did you mean to exclude operator and punctuation?

File: src/styles/srcery.css [hunk 1]
```diff
@@ -2,76 +2,92 @@
...
+/* Bright black */
+.hljs-link,
+.hljs-number,
 .hljs-comment,
-.hljs-deletion,
 .hljs-meta {
   color: #918175;
 }
+
+.hljs,
```
Comment (joshgoebel) [inline src/styles/srcery.css:76]: This looks incorrect.  `hljs-subst` should perhaps get BG and FG applied, but not different layout attributes.  Should `subst` be split out with just the color and background?

Comment (redguardtoo) [reply src/styles/srcery.css]: @joshgoebel , thanks for the review. I force pushed the latest code.

I didn't do any radical change. I follow the Emacs implementation of same theme and make some small adjustment.

**Comment by joshgoebel** [comment]
I added some bits to the header and removed the "default" boilerplate (since the build process adds that now).

**Pull request merged**